### DISCUSSION TITLE
What is HTML?

### DISCUSSION THREAD

**Discussion by xlgmokha** [CodeChica/plus-plus#30] [General]
I have trouble remember what HTML is and what's is used for. Can you remind me what is HTML?

**Comment by daitanfutekina** [comment]
i mean like, you are our teacher so I hope not

**Comment by daitanfutekina** [comment]
ok, html is the primary structure everything is connected to, and you'd connect the css and java to it to make it have a use or appearance, other than that it can show text and other basic stuff and also has ways to comment and structure in the code which allows you keep it organized

**Reply by xlgmokha** [reply]
Awesome, description. One thing that I would like to adjust is that Java is a different programming language from JavaScript. 

[Java](https://docs.oracle.com/javase/specs/jls/se8/html/index.html) and [JavaScript](https://www.javascript.com/) have a similar name but operate in slightly different ways/environments. The names confused me a lot when I was younger. I hope this helps.

**Comment by superrrk** [comment]
HTML (HyperText Markup Language) is a language that is used to outline the content in a web browser. While writing this code, you're able to organize the content into containers known as divs and other organization tags, which is super helpful when it comes to styling the content later on in CSS or referencing it in JavaScript.

**Comment by IxchelO** [comment]
HTML is the base code where everything starts out from. You add CSS and JavaScript to it afterwards.

**Comment by Ariadnaqlla** [comment]
the primary base everything is connected too

**Comment by NeftalynBarrera** [comment]
I think html is the overall base that we use to put our coding in.

**Comment by amisadaib** [comment]
Html is were we put our coding in and it puts everything together.

**Comment by code2417** [comment]
HTML. which stands for Hypertext Markup Language, is the standard markup language for creating webpages. With the help of HTML, users can view the webpage on the internet. HTML opens up to other languages like CSS and JavaScript.

**Comment by IxchelO** [comment]
HTML is where most of our code starts off of and is the basis of our code.
